book/modules/appendix1.tex

```diff
@@ -268,7 +305,7 @@
 	\]
 	We call the matrix an \emph{augmented matrix} to stress that it contains two types of information:
 	the \emph{coefficients} of the variables $t$, $s$, and $r$ and the \emph{constants} on
-	the right hand side of equation. An (optional) vertical line
```

Comment:
I'm not sure if I changed the meaning of this sentence in the wrong way...

Reply:
Change it back. But make `equation` -> `the equations`